build/blobplugin/Dockerfile

```diff
@@ -18,10 +18,13 @@ ARG ARCH=amd64
 ARG binary=./_output/${ARCH}/blobplugin
 COPY ${binary} /blobplugin
 
+COPY ./build/blobplugin/blobfuse-proxy /blobfuse-proxy
```

Comment:
would you keep the original path: `pkg/blobplugin`? that's what other CSI drivers are like.

Reply:
I move the dockerfile and package configurations to `build` directory according to the suggestion from  https://github.com/golang-standards/project-layout, looks this can make the project layout more clear.

Reply:
o, I just realized that our MCR image build depends on this Dockerfile path, it's better not change the path, otherwise MCR image build should have two Dockerfile paths for new and old path versions, let's keep as it.

Reply:
OK, that sounds reasonable.